### PR TITLE
APM-43: Migration Glue name changed to Acquia Platform Migration(APM).

### PR DESCRIPTION
**Pull request by gulabbisht**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# migration_glue
+# apm
```
Comment (joshirohit100) [inline README.md]: This should be `Acquia Platform Migration`

File: apm.info.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Migration Glue
+name: Acquia Platform Migration (APM)
 type: module
 description: 'Glue between migrate_tools and migration_mapper.'
```
Comment (joshirohit100) [inline apm.info.yml]: I think we also need to update the description (with what - need to discuss) cc @amitgoyal

**Comment by joshirohit100** [comment]
Module name is changed from `migration_glue` => `apm` but new name `apm` is not looking right. Should it be `acquia_platform_migration`? cc @amitgoyal

**Comment by gulabbisht** [comment]
@joshirohit100 I would also prefer to use `acquia_platform_migration` in the code.

**Comment by gulabbisht** [comment]
Raising another PR.

**Pull request closed without merging**